content/en/docs/tasks/security/health-check/index.md

```diff
@@ -20,7 +20,7 @@ If the file is not updated for a period, Kubelet will restart the Citadel contai
 
 Note: because Citadel health checking currently only monitors the health status of CSR service API,
 this feature is not needed if the production setup is not using the
```

Comment:
Is the word 'the' grammatically correct here?

Reply:
Probably a legacy from earlier versions, thanks for the catch.